src/Button/ZoomToExtentButton/ZoomToExtentButton.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import OlMap from 'ol/map';
+import OlExtent from 'ol/extent';
+import OlSimpleGeometry from 'ol/geom/simplegeometry';
+import {
+  SimpleButton
+} from '../../index';
+import { TestUtil } from '../../Util/TestUtil';
```

Comment:
Do you really need the `TestUtil` class here? It will be mostly used only in test classes..

Reply:
:+1: Plz remove